### ISSUE TITLE
JSX-a11y error when running `npm run dev`

### ISSUE THREAD

**Issue by pjlee11** [bbc/simorgh#596]
**Describe the bug**
There is an error across multiple files in the application that throws:

```
Definition for rule 'jsx-a11y/label-has-associated-control' was not found  jsx-a11y/label-has-associated-control
```
The files the error appears to be resonating from do not in anyway use HTML label logic and therefore this seems to be an error saying this rule has not yet been defined. 

**To Reproduce**
1. `git checkout da709cdd`
2. `rm -rf node_modules package-lock.json && npm run dev`
3. Open http://localhost:7080 in a browser

**Expected behavior**
Should not error on dev build.

The rule was turned off in [this PR](https://github.com/BBC-News/simorgh/pull/597). We should investigate whether this is an issue if the rule is enabled, and how to fix it.

**Screenshots**
![image](https://user-images.githubusercontent.com/7791726/44976802-36719480-af5e-11e8-8b5f-b082e1fa008e.png)

**Additional context**
To be concise the error being thrown is due to a lack of rule declaration not the rule failing to be met. Also this only affects the dev build as production released and passes tests as expected.
